### ISSUE TITLE
New executable is not generated after clean

### ISSUE THREAD

**Issue by tupton** [gogradle/gogradle#279]
I read #217 but I seem to still experience the issue where the executable isn't created after a previous build and a clean. In this example I'm using a target platform, but the issue exists even without it.

```gradle
plugins {
    id 'com.github.blindpirate.gogradle' version '0.11.2'
}

apply plugin: 'base'

golang {
    packagePath = 'github.com/your/package' // go import path of project to be built, NOT local file system path!
    goVersion = '1.10'
}

goBuild {
    targetPlatform = ['linux-amd64']
    outputLocation = "${project.buildDir}" + '/myproject${GOEXE}'
}

clean {
    delete project.buildDir
}

```

The following commands show the presence of the built executable, a clean, an attempt to rebuild that shows the task still up-to-date, and a forced rebuild with `--rerun-tasks`.

I have a feeling I'm missing something obvious about getting gradle to rebuild the executable here.

```
❯ l build
total 3968
drwxr-xr-x   3 tupton  staff    96B Feb  6 16:01 ./
drwxr-xr-x  13 tupton  staff   416B Feb  6 16:01 ../
-rwxr-xr-x   1 tupton  staff   1.9M Feb  6 16:01 myproject*
❯ ./gradlew clean goClean

BUILD SUCCESSFUL in 0s
2 actionable tasks: 2 executed
❯ l build
ls: build: No such file or directory
❯ ./gradlew goBuild --console plain

> Task :goPrepare
Use project GOPATH: /path/scrubbed/gogradle-no-rebuild/.gogradle/project_gopath

> Task :resolveBuildDependencies
> Task :installDependencies
> Task :buildLinuxAmd64 UP-TO-DATE
> Task :goBuild

BUILD SUCCESSFUL in 0s
5 actionable tasks: 4 executed, 1 up-to-date
❯ l build
ls: build: No such file or directory
❯ ./gradlew goBuild --console plain --rerun-tasks

> Task :goPrepare
Use project GOPATH: /path/scrubbed/gogradle-no-rebuild/.gogradle/project_gopath

> Task :resolveBuildDependencies
> Task :installDependencies
> Task :buildLinuxAmd64
> Task :goBuild

BUILD SUCCESSFUL in 0s
5 actionable tasks: 5 executed
❯ l build
total 3968
drwxr-xr-x   3 tupton  staff    96B Feb  6 16:02 ./
drwxr-xr-x  13 tupton  staff   416B Feb  6 16:02 ../
-rwxr-xr-x   1 tupton  staff   1.9M Feb  6 16:02 myproject*
```

**Comment by blindpirate** [comment]
Sorry I was on vacation last week. It looks like something wrong. I'll investigate and fix it this week.

**Comment by blindpirate** [comment]
Fixed by 0.11.3.
